python/python/lance/dataset.py

```diff
@@ -463,8 +464,10 @@ def _create_version_from_fragments(
         -----
         This method is for internal use only.
         """
-        ds = self._ds.create_version_from_fragments(new_schema, fragments)
-        return LanceDataset(self.uri)
+        if isinstance(base_uri, Path):
```

Comment:
i forgot, do we need to do any relative to absolute conversion or $HOME expansion etc here? or is that all done at the Rust level?

Reply:
i am not sure which side does the normalization now.